src/modules/profile/profile.service.ts

```diff
@@ -14,34 +15,31 @@ export class ProfileService {
   ) {}
 
   async findOneProfile(userId: string) {
-    try {
-      const user = await this.userRepository.findOne({ where: { id: userId } });
-      if (!user) {
-        throw new NotFoundException('User not found');
-      }
-
-      const userProfile = await this.userRepository.findOne({
-        where: { id: userId },
-        relations: ['profile'],
-      });
+    if (!isUUID(userId, '4')) {
+      throw new BadRequestException('Invalid user ID');
```

Comment:
Apply the global custom exception error handler

Reply:
Done